SCRIPTING

```diff
@@ -78,9 +80,13 @@ Running
   one solution is to pre-generate all requests in init() and do a quick
   lookup in request().
 
-  response() is called with the HTTP response status, headers, and body.
-  Parsing the headers and body is expensive, so if the response global is
+  response() is called with the HTTP response status and headers.
+  Parsing the headers is expensive, so if the response global is
   nil after the call to init() wrk will ignore the headers and body.
+  Using wrk's --lua-dont-pass-body command line option will cause a "dummy"
+  body to be parsed instead of the actual body. The option is useful e.g. for
+  benchmarking endpoints which pass large response bodies, like caching
```

Comment:
The "like caching server" part seems out of place. You could be benchmarking a caching server that caches small responses. I suggest deleting that last part.

Reply:
Makes sense, will remove the caching server reference.